install/update_080_0801.php

```diff
@@ -38,39 +38,44 @@
 function update080to0801() {
    global $DB, $migration;
 
-   $updateresult = true;
-
    //TRANS: %s is the number of new version
    $migration->displayTitle(sprintf(__('Update to %s'), '0.80.1'));
    $migration->setVersion('0.80.1');
 
    // Clean duplicates
-   $query = "SELECT COUNT(*) AS CPT, `tickets_id`, `type`, `groups_id`
-             FROM `glpi_groups_tickets`
-             GROUP BY `tickets_id`, `type`, `groups_id`
-             HAVING CPT > 1";
-   if ($result=$DB->query($query)) {
-      if ($DB->numrows($result)>0) {
-         while ($data=$DB->fetch_array($result)) {
-            // Skip first
-            $query = "SELECT `id`
-                      FROM `glpi_groups_tickets`
-                      WHERE `tickets_id` = '".$data['tickets_id']."'
-                            AND `type` = '".$data['type']."'
-                            AND `groups_id` = '".$data['groups_id']."'
-                      ORDER BY `id` DESC
-                      LIMIT 1,99999";
-            if ($result2=$DB->query($query)) {
-               if ($DB->numrows($result2)) {
-                  while ($data2=$DB->fetch_array($result2)) {
-                     $query = "DELETE
-                               FROM `glpi_groups_tickets`
-                               WHERE `id` ='".$data2['id']."'";
-                     $DB->queryOrDie($query, "0.80.1 clean to update glpi_groups_tickets");
-                  }
-               }
-            }
-         }
+   $query = [
+      'SELECT'    => [
+         "tickets_id",
+         "type",
+         "groups_id"
+      ],
+      'COUNT'     => "CPT",
```

Comment:
I'm not sure this one will give expected results, this should be checked to be sure